oval_graph/oval_node.py

```diff
@@ -19,9 +19,7 @@ class OvalNode():
     """
 
     def __init__(self, **kwargs):
```

Comment:
Just curious - why the function accepts only kwargs, wile the comment suggests that there are required arguments?